### PR TITLE
 Fix wrong property for the root path of the schema server

### PR DESCRIPTION
**Pull request by making**

### CODE REVIEW

File: spring-cloud-stream-schema-server/src/main/java/org/springframework/cloud/stream/schema/server/controllers/ServerController.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
  * @author Vinicius Carvalho
  */
 @RestController
-@RequestMapping(path = "${spring.cloud.stream.schema.server.prefix:}")
```
Comment (mbogoevici) [inline spring-cloud-stream-schema-server/src/main/java/org/springframework/cloud/stream/schema/server/controllers/ServerController.java]: Not very clear what is the intent here? Just changing the name of the property?

Comment (making) [inline spring-cloud-stream-schema-server/src/main/java/org/springframework/cloud/stream/schema/server/controllers/ServerController.java]: `spring.cloud.stream.schema.server.path` is mentioned in the [doc](
http://docs.spring.io/spring-cloud-stream/docs/Brooklyn.SR1/reference/htmlsingle/#_schema_registry_server) and defined in [`SchemaServerProperties`](https://github.com/making/spring-cloud-stream/blob/6d8cb58bdefa3ab49d562954b652e61a02f84aa5/spring-cloud-stream-schema-server/src/main/java/org/springframework/cloud/stream/schema/server/config/SchemaServerProperties.java#L33) but unavailable.

Comment (mbogoevici) [inline spring-cloud-stream-schema-server/src/main/java/org/springframework/cloud/stream/schema/server/controllers/ServerController.java]: Gotcha, thanks for catching this!

**Comment by ilayaperumalg** [comment]
LGTM, merged.

**Pull request merged**